security/pinact/Portfile

```
@@ -0,0 +1,174 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/suzuki-shunsuke/pinact 2.2.0 v
+categories          security
+maintainers         {macports.halostatue.ca:austin @halostatue} \
+                    openmaintainer
+license             MIT
+
+description         A CLI to edit GitHub Workflow and Composite action files and pin \
+                    versions of Actions and Reusable Workflows. pinact can also update \
+                    their versions and verify version annotations.
+
+long_description    {*}${description}
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  376eaf6e2d61ca072f6142ee5d1cddae02cbecf1 \
+                    sha256  047745c193add771fb49f16dcfa49f7f186eb74b510296faf49b33a04a760090 \
+                    size    33143
+
+build.args-append   -ldflags \"-X main.version=${version} -X main.commit=MacPorts \\" \
```

The last double quote is not being properly escaped.

```suggestion
build.args-append   -ldflags \"-X main.version=${version} -X main.commit=MacPorts \" \
```